vlm/src/main/scala/geotrellis/contrib/vlm/geotiff/GeoTiffResampleRasterSource.scala

```diff
@@ -32,13 +32,13 @@ case class GeoTiffResampleRasterSource(
   method: ResampleMethod = NearestNeighbor,
   strategy: OverviewStrategy = AutoHigherResolution,
   private[vlm] val targetCellType: Option[TargetCellType] = None,
-  private[vlm] val baseTiff: Option[MultibandGeoTiff] = None
+  @transient private[vlm] val baseTiff: Option[MultibandGeoTiff] = None
```

Comment:
Since `baseTiff` is transient now, to prevent serialization issues; the following case can happen on shuffle: 

```scala
val baseTiff: Option[MultibandGeoTiff] = null
Option(baseTiff).flatten //> None

```